nilearn/masking.py

```diff
@@ -254,8 +254,6 @@ def compute_epi_mask(epi_img, lower_cutoff=0.2, upper_cutoff=0.85,
     if verbose > 0:
         print("EPI mask computation")
 
-    epi_img = _utils.check_niimg(epi_img)
```

Comment:
This is clever but your efforts are kinda wasted by the `new_img_like` at line 289/287.